src/applications/facility-locator/tests/e2e/providerSearch.cypress.spec.js

```diff
@@ -11,6 +11,41 @@ describe('Provider search', () => {
     cy.intercept('GET', '/geocoding/**/*', mockGeocodingData);
   });
 
+  it('renders "Search for available service" prompt', () => {
+    cy.visit('/find-locations');
+    cy.injectAxe();
```

Comment:
I think this can be removed since we aren't running an axe check.

Reply:
removed